main.go

```diff
@@ -34,24 +34,33 @@ func main() {
 
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/" {
+			http.NotFound(w, r)
+			return
+		}
 		w.WriteHeader(200)
 		_, err = w.Write(indexPage)
 		if err != nil {
 			log.Fatal(err)
 		} else {
-			log.Printf(fmt.Sprintf("GET - / 200 from %s", r.UserAgent()))
+			log.Printf(fmt.Sprintf("GET - / 200 UserAgent: %s", r.UserAgent()))
 		}
 	})
-	mux.HandleFunc("/swagger.json", func(w http.ResponseWriter, r *http.Request) {
+	mux.HandleFunc("/swagger.json/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
not sure we want a trailing slash on this
```suggestion
	mux.HandleFunc("/swagger.json", func(w http.ResponseWriter, r *http.Request) {
```